.yarnrc.yml

```diff
@@ -2,4 +2,8 @@ enableGlobalCache: true
 
 nodeLinker: node-modules
 
-yarnPath: .yarn/releases/yarn-berry.js
+plugins:
+  - path: .yarn/plugins/@yarnpkg/plugin-interactive-tools.cjs
+    spec: "@yarnpkg/plugin-interactive-tools"
+
+yarnPath: .yarn/releases/yarn-2.3.3.cjs
```

Comment:
can we do this on master as well?

Do note that we need "sources" for now, due to https://github.com/yarnpkg/berry/issues/1983

Reply:
> ESLint is failing due to missing resolvers from `eslint-plugin-import`
> 
> I'll add the missing eslint settings and the [required peer dependency](https://github.com/benmosher/eslint-plugin-import#typescript) `@typescript-eslint/parser`

Ok funny thing: it's failing not due to `eslint-plugin-imports` settings, but rather babel [exclude `.ts` files by default in compilation](https://babeljs.io/docs/en/babel-preset-typescript)

I'll add the `-x ".ts"` flag to the babel scripts😅

Reply:
Sure thing, I'll do a separate PR (probably by tomorrow).

But can you clarify what do you mean by "sources"?
Does this mean I have to run `yarn set version from sources --branch 1985`?

Reply:
I have reverted the yarn executable and plugin installation in this pr, so the yarn sources issue would not be a blocker to merge this pr🙂